### ISSUE TITLE
Request for Training Code

### ISSUE THREAD

**Issue by xyliugo** [HKUST-LongGroup/CoMM#3]
Thank you for the great work on the dataset and evaluation benchmark! I noticed the model was fine-tuned by your team, and I was wondering if you could share the training code, even if it's still raw. It would be incredibly helpful. Thanks again!

**Comment by chenwei746** [comment]
Hi,
I have release the training code of MiniGPT-5 and SEED-Llama.
